### ISSUE TITLE
Quickstart eslint example may be misleading

### ISSUE THREAD

**Issue by samwagg** [flycheck/flycheck#1813]
### TLDR
I think the eslint installation command in the quickstart should be updated to include the `--global` flag, since flycheck uses whatever `eslint` executable is on the PATH by default and `npm install eslint` installs in `node_modules` in the current directory.

### A few more details

I followed the [quickstart](https://www.flycheck.org/en/latest/user/quickstart.html) instructions using `npm` and `eslint` and `flycheck` wasn't able to discover my eslint executable. I think I understand now that flycheck uses whatever executable is on your PATH and does not have any special logic for discovering the executable in local `node_modules`. However, I think by instructing the user to install eslint as a project-level node module, this documentation is misleadingly implying that such discovery is supported.
